_migration-assistant/deploying-migration-assistant/getting-started-data-migration.md

```diff
@@ -118,7 +118,12 @@ These commands deploy the following stacks:
 
 Use the following steps to configure and deploy RFS, deploy Migration Assistant, and verify installation of the required stacks:
 
-1. Add the source and target cluster password as separate **Secrets** in [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) as an unstructured string. Be sure to copy the secret Amazon Resource Name (ARN) for use during deployment.
+1. Add the basic auth details (username/password) for the source and target cluster as separate **Secrets** in [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html). Each secret should contain two Key/value pairs, one for username and one for password. This will result in the secret Plaintext looking like the example below.
```

Comment:
* Remove Secrets cap and bold type
* Lowercase "Key"
* Lowercase "Plaintext

Reply:
Updated thanks!